src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -134,4 +158,21 @@ private function convertToBytes(string $memoryLimit): int
 
         return $max;
     }
+
+    private function findAlternatives($name, array $collection)
```

Comment:
Isn't this already somewhere else? Can't we share instead of copy? Maybe at least reference the other occurrences if it has been pasted so we know when maintaining it?

Reply:
Yes, from debug command of the Form component, but still there is other places where similar procedure has been used, probably a `trait` could be created for such cases.

Reply:
We don"t have a central place for such logic for 2 reasons: it's small enough and we don't want to create a "util" component.

Reply:
@fabpot a good practice can be to add a comment that refers to the place it was copy/pasted from.